plugin-dev/Source/Sentry/Private/HAL/PlatformSentryAttachment.h

```diff
@@ -1,21 +1,21 @@
-// Copyright (c) 2025 Sentry. All Rights Reserved.
-
-#pragma once
-
-#if PLATFORM_ANDROID
-#include "Android/AndroidSentryAttachment.h"
-#elif PLATFORM_APPLE
-#include "Apple/AppleSentryAttachment.h"
-#else
-#include "Null/NullSentryAttachment.h"
-#endif
-
-static TSharedPtr<ISentryAttachment> CreateSharedSentryAttachment(const FString& Path, const FString& Filename, const FString& ContentType)
-{
-	return MakeShareable(new FPlatformSentryAttachment(Path, Filename, ContentType));
-}
-
-static TSharedPtr<ISentryAttachment> CreateSharedSentryAttachment(const TArray<uint8>& Data, const FString& Filename, const FString& ContentType)
-{
-	return MakeShareable(new FPlatformSentryAttachment(Data, Filename, ContentType));
-}
+// Copyright (c) 2025 Sentry. All Rights Reserved.
+
+#pragma once
+
+#if PLATFORM_ANDROID
+#include "Android/AndroidSentryAttachment.h"
+#elif PLATFORM_APPLE
+#include "Apple/AppleSentryAttachment.h"
+#else
+#include "Null/NullSentryAttachment.h"
+#endif
+
+static TSharedPtr<ISentryAttachment> CreateSharedSentryAttachment(const FString& Path, const FString& Filename, const FString& ContentType)
+{
+	return MakeShareable(new FPlatformSentryAttachment(Path, Filename, ContentType));
```

Comment:
also, any reason to use tabs instead of spaces?

Reply:
That's a [coding standard](https://dev.epicgames.com/documentation/en-us/unreal-engine/epic-cplusplus-coding-standard-for-unreal-engine#tabsandindenting) for Unreal.